agent/statemanager/state_manager_windows.go

```diff
@@ -0,0 +1,139 @@
+// +build windows
+
+// Copyright 2014-2016 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//	http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package statemanager
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/cihub/seelog"
+
+	"golang.org/x/sys/windows/registry"
+)
+
+const (
+	ecsDataFileRootKey   = registry.LOCAL_MACHINE
+	ecsDataFileKeyPath   = `SOFTWARE\Amazon\ECS Agent\State File`
```

Comment:
Can we get rid of the space character here in the [commit](b23f949d5bb0c1345ef7f9bf3e039067c0ade0e2)?